main/2023/CG-10.md

```diff
@@ -51,48 +51,54 @@ After entering the inner yard, turn right in the yard and go to the reception wh
 #### Meals
 There will likely be some catering provided by Google. Details are to be defined and will be posted here.
 
-### Proposed agenda items (to be scheduled)
+### Schedule
 
-Please add any scheduling constraints here, especially if you're not able to travel, but would like to contribute to one of the agenda items. If you are adding a scheduling constraint, add the agenda item you are interested in attending, and timeslots that would work. Adding a constraint doesn't guarantee that we will be able to accommodate it, but we will attempt to make sure interested folks can attend and contribute. 
-
-- ...
+**Wednesday, 11th October**
+- Multi-Memory Phase 4 Vote
+  - 15 mins (Andreas Rossberg, Ashley Nelson)
+- Extended Name Section: Update and discussion on [name declarations](https://github.com/WebAssembly/annotations/issues/21)
+  - 15 mins (Ashley Nelson)
+- Profiles: Introducing new profiles, artifacts, and procedural discussion
+  - 60 mins (Andreas Rossberg)
+- Relaxed SIMD: Phase 4 vote
+  - 10 mins (Deepti Gandluri)
+- Type Reflection for WebAssembly JavaScript API: Update, tentative poll for Phase 4
+  - 30 mins (Ilya Rezvov)
+- Exception handling: Vote on [the proposed spec for reintroducing exnref](https://github.com/WebAssembly/exception-handling/issues/281)
+  - 30 mins (Heejin Ahn, Ben Titzer) 
+- Initial work on a DSL for the specification document
+  - 40 mins (Andreas Rossberg, Sukyoung Ryu, Conrad Watt)
+  - Prefer Wednesday, October 11th
 - Update on [threads proposal](https://github.com/WebAssembly/threads) (with phase 4 vote?)
```

Comment:
Do we know yet whether we expect a phase 4 vote?
(if not, maybe this could even just say "with possible phase 4 vote")

Reply:
Updated in the doc.